dashboard/src/features/tutorial/components/Success/Success.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styles from './Success.scss'
+import { Link } from 'react-router'
+
+class Success extends React.Component {
```

Comment:
I think `Success` and `Description` are similar enough as to share code and styles with a little refactoring.